### PR TITLE
Trader/config values

### PR DESCRIPTION
**Pull request by fbielejec**

### CODE REVIEW

File: relayer/relayer/src/config.rs [hunk 1]
```diff
@@ -53,6 +53,24 @@ pub struct Config {
...
+    #[arg(long, default_value = "100_000_000_000_000")]
+    pub eth_to_azero_relaying_buffer: u128,
+
+    /// Trader component will bridge azero wETH after the balance exceeds this amount
+    ///
+    /// Defaults to 0.1 ETH
+    #[arg(long, default_value = "100_000_000_000_000_000")]
+    pub bridging_threshold: u128,
+
+    /// Trader component will request relaying rewards when they exceed this value
```
Comment (krzysztofziobro) [inline relayer/relayer/src/config.rs:68]:
```suggestion
    /// Trader component will claim rewards when they exceed this value
```

File: relayer/relayer/src/config.rs [hunk 2]
```diff
@@ -53,6 +53,24 @@ pub struct Config {
     )]
     pub run_trader_component: bool,
 
+    /// Trader component will always keep this + current base_fee in the guardians balance and will only sell the surplus
+    ///
+    /// Defaults to 100 AZERO
+    #[arg(long, default_value = "100_000_000_000_000")]
```
Comment (krzysztofziobro) [inline relayer/relayer/src/config.rs:59]: Let's set the default value to 1000 AZERO

Comment (fbielejec) [reply relayer/relayer/src/config.rs]: ok

File: relayer/relayer/src/config.rs [hunk 3]
```diff
@@ -53,6 +53,24 @@ pub struct Config {
     )]
     pub run_trader_component: bool,
 
+    /// Trader component will always keep this + current base_fee in the guardians balance and will only sell the surplus
+    ///
+    /// Defaults to 1000 AZERO
+    #[arg(long, default_value = "1000_000_000_000_000")]
```
Comment (krzysztofziobro) [inline relayer/relayer/src/config.rs:59]:
```suggestion
    #[arg(long, default_value = "1_000_000_000_000_000")]
```

**Review by krzysztofziobro — APPROVED** [review]

**Pull request merged**